turbopack/crates/turbopack-dev-server/src/update/stream.rs

```diff
@@ -87,13 +87,13 @@ impl GetContentFn {
     }
 }
 
-async fn peek_issues<T: Send>(source: OperationVc<T>) -> Result<Vec<ReadRef<PlainIssue>>> {
```

Comment:
I really wonder why you changed the plain issue conversion to be not a turbo-tasks function and use `PlainIssue` instead of `ReadRef<PlainIssue>`.
Being able to cache `into_plain` sounds like a good idea to me. Also `Vec<ReadRef<PlainIssue>>` sounds cheaper than `Vec<PlainIssue>`. Note that we have to run these conversion on every HMR change, so this could affect HMR performance... Especially with many warnings in the compilation.

Reply:
the logic for formatting issues to PlainIssue should be extremely cheap so caching it seemed wasteful and like it would introduce complexity.

i didn't really consider the HMR usecase but even there it really seems like formatting should be very fast so who cares?  Still i can add it back, it just intuitively struck me as wasteful to cache issue formatting especially given that we don't have an eviction strategy.